arrow-array/src/arithmetic.rs

```diff
@@ -45,61 +45,83 @@ pub trait ArrowNativeTypeOp: ArrowNativeType {
     /// The multiplicative identity
     const ONE: Self;
 
+    /// A checked add operation
```

Comment:
```suggestion
    /// Checked addition operation
```

`A` seems redundant to me.